game/scripts/vscripts/components/filters/statusresistance.lua

```diff
@@ -30,7 +30,7 @@ function StatusResistance:StatusResistanceFilter(filterTable)
       if modifier.GetModifierStatusResistanceStacking and modifier:GetModifierStatusResistanceStacking(params) then
         stackResist = (stackResist or 0) + modifier:GetModifierStatusResistanceStacking(params)
       end
-      if modifier.GetModifierStatusResistance and modifier:GetModifierStatusResistance(params) and modifier:GetModifierStatusResistance(params) > resistance then
+      if modifier.GetModifierStatusResistance and modifier:GetModifierStatusResistance(params) and math.abs(modifier:GetModifierStatusResistance(params)) > math.abs(resistance) then
```

Comment:
i think this math.abs isn't right here, really non-stacking status resistance should never be negative, it only makes sense with stacking...